src/menu_base.cpp

```diff
@@ -337,19 +337,56 @@ std::string MenuBase::processGenericScreens()
 /**
  * @brief Setup light bar string, and return default display.
  */
-std::string MenuBase::setupYesNoMenuInput()
+std::string MenuBase::setupYesNoMenuInput(const std::string &menu_prompt, std::vector<MapType> &code_map)
 {
     m_input_index = MENU_YESNO_BAR;
+    clearMenuPullDownOptions();
+
+    // Then feed though and return the updated string.
+    std::string prompt_string = std::move(m_session_io.parseCodeMapGenerics(menu_prompt, code_map));
+    std::string display_prompt = moveStringToBottom(prompt_string);
 
-    std::string output = "-Yes- -No-  (Mockup)";
+    std::string yesNoBars = "\x1b[0m|01\x1b[1;37;41m%01\x1b[0m \x1b[0m|02\x1b[1;37;41m%02\x1b[0m";
+    yesNoBars.insert(0, display_prompt);
 
-    // Generate y/n light bar commands
-    // dependon on default m_active_pulldownID for y/n
+    // Parse the Screen to the Screen Buffer.
+    m_ansi_process->parseAnsiScreen((char *)yesNoBars.c_str());
 
-    // decide if we want to buld string on default colors
-    // or parse an extra template from config??!?
+    // Screen to String so it can be processed.
+    m_ansi_process->screenBufferToString();
 
-    return output;
+    // Process buffer for PullDown Codes.
+    // only if we want result, ignore.., result just for testing at this time!
+    std::string result = std::move(m_ansi_process->screenBufferParse());
+
+    // Update Lightbars, by default they have no names for YES/NO/Continue prompts.
+    for(unsigned int i = 0; i < m_menu_info->menu_options.size(); i++)
```

Comment:
Section only handles Yes / No,  Yes / No / Continue will be added.

Also the default colors are set to White on Red Background,  forgot to change this to the current theme colors, update this later on!